Library/Homebrew/cmd/missing.rb

```diff
@@ -6,15 +6,40 @@ module Homebrew
   def missing
     return unless HOMEBREW_CELLAR.exist?
 
-    ff = if ARGV.named.empty?
+    formulae = if ARGV.named.empty?
       Formula.installed
     else
       ARGV.resolved_formulae
     end
+    if ARGV.json == "v1"
+      missing = print_missing_json(formulae)
+    else
+      missing = print_missing(formulae)
+    end
+    Homebrew.failed = ARGV.resolved_formulae.any? && missing.any?
```

Comment:
Correct. Or to phrase it slightly differently, it makes sense to me that the exit status for `brew missing` and `brew missing $(brew list)` is consistent.